src/test/resources/logs/getLogsNodeService.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Copyright (c) 2024, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of the OperatorFabric project.
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+url=$3
+if [[ -z $url ]]
+then
+	url="http://localhost"
+fi
+if [[ -z $2 ]]
+then
+    echo "Usage : getLogsNodeService.sh host:port user opfab_url"
+else
+	serviceAddress=$1
+	user=$2
+	
+	source ../getToken.sh $url $user
+	
+	response=$( curl -s "http://$serviceAddress/logLevel" -H "Authorization: Bearer $token" -H "Content-type:application/json")
```

Comment:
This script is based on existing script for java services .
I realized that it is not possible to set a password , could you add this possibility for the existing and for the new scripts 

`Usage : getLogsNodeService.sh host:port user passwd opfab_url`

With default value test